configure.ac

```diff
@@ -2400,7 +2411,6 @@ CheckGLX()
         AC_MSG_RESULT($video_opengl_glx)
         if test x$video_opengl_glx = xyes; then
             AC_DEFINE(SDL_VIDEO_OPENGL, 1, [ ])
-            AC_DEFINE(SDL_VIDEO_OPENGL_GLX, 1, [ ])
```

Comment:
Why are you removing this? `SDL_VIDEO_OPENGL_GLX` isn't defined elsewhere either, so it is needed.

Reply:
Mistake, reverted